src/modules/assignment-4/questions/question10/Task10.jsx

```diff
@@ -0,0 +1,87 @@
+import { Button, FormControl, FormGroup, TextField } from "@mui/material";
+import "../Styles.css";
+
+import * as yup from "yup";
+import { useFormik } from "formik";
+
+const Task10 = () => {
+  const validationSchema = yup.object({
+    username: yup.string().required("Name is required").min(6),
+    age: yup.number().positive().integer(),
+    password: yup.string().required("Password is required").min(8),
+    email: yup.string().email(),
+  });
+
+  const formik = useFormik({
+    initialValues: {
+      username: "",
+      age: null,
+      email: "",
+      password: "",
+    },
+    onSubmit: (values) => {
+      console.log(values);
+    },
+    validationSchema: validationSchema,
+  });
+
```

Comment:
![image](https://github.com/ayush1-successive/react-training/assets/83696224/3d2fb102-659a-416d-a355-8420554e3874)
password should not be visible